doc/source/whatsnew/v1.1.2.rst

```diff
@@ -51,7 +51,7 @@ Bug fixes
 Other
 ~~~~~
 - :meth:`factorize` now supports ``na_sentinel=None`` to include NaN in the uniques of the values and remove ``dropna`` keyword which was unintentionally exposed to public facing API in 1.1 version from :meth:`factorize` (:issue:`35667`)
-- :meth:`DataFrame.plot` and meth:`Series.plot` raise ``UserWarning`` about usage of FixedFormatter and FixedLocator (:issue:`35684` and :issue:`35945`)
+- :meth:`DataFrame.plot` and :meth:`Series.plot` raise ``UserWarning`` about usage of FixedFormatter and FixedLocator (:issue:`35684` and :issue:`35945`)
```

Comment:
Should FixedFormatter and FixedLocator have double backticks?

Reply:
too bad the docs build doesn't warn on these things. maybe can have some sort of code check (at least to check the book-ended meth/func and so on

Reply:
> Should FixedFormatter and FixedLocator have double backticks?

not sure. they are mpl classes. added anyway.

may have been better to include the message verbatim in this case instead `UserWarning: FixedFormatter should only be used together with FixedLocator` as was done for `ValueError: buffer source array is read-only` above.